python/lsst/summit/extras/plotting/psfPlotting.py

```diff
@@ -0,0 +1,524 @@
+        The figure object to plot on.
+    axes : `numpy.ndarray`
+        The array of axes objects to plot on.
+    table : `numpy.ndarray`
+        The table containing the data to be plotted.
+    camera : `list`
+        The list of camera detector objects.
+    maxPoints : `int`, optional
+        The maximum number of points to plot. If the number of points in the
+        table is greater than this value, a random subset of points will be
+        plotted.
+    saveAs : `str`, optional
+        The file path to save the figure.
+    """
+    table = selectPoints(table, maxPoints)
+
+    cbar = addColorbarToAxes(axes[0, 0].scatter(table["nw_x"], table["nw_y"], c=table["T"], s=5))
+    cbar.set_label("T [arcsec$^2$]")
+
+    emax = np.quantile(np.abs(np.concatenate([table["e1"], table["e2"]])), 0.98)
+    cbar = addColorbarToAxes(
+        axes[1, 0].scatter(
+            table["nw_x"], table["nw_y"], c=table["nw_e1"], vmin=-emax, vmax=emax, cmap="bwr", s=5
+        )
+    )
+    cbar.set_label("e1")
+
+    cbar = addColorbarToAxes(
+        axes[1, 1].scatter(
+            table["nw_x"], table["nw_y"], c=table["nw_e2"], vmin=-emax, vmax=emax, cmap="bwr", s=5
+        )
+    )
+    cbar.set_label("e2")
+
+    Q = axes[0, 1].quiver(
+        table["nw_x"],
+        table["nw_y"],
+        table["e"] * np.cos(0.5 * np.arctan2(table["nw_e2"], table["nw_e1"])),
+        table["e"] * np.sin(0.5 * np.arctan2(table["nw_e2"], table["nw_e1"])),
+        headlength=0,
+        headaxislength=0,
+        scale=1,
+        pivot="middle",
+    )
+    axes[0, 1].quiverkey(Q, X=0.08, Y=0.95, U=0.05, label="0.05", labelpos="S")
+
+    for ax in axes.ravel():
+        ax.set_xlabel("West")
+        ax.set_ylabel("North")
+        ax.set_aspect("equal")
+        ax.set_xlim(-90, 90)
+        ax.set_ylim(-90, 90)
+
+    # Plot camera detector outlines
+    nwRot = table.meta["nwRot"]
+    for det in camera:
+        xs = []
+        ys = []
+        for corner in det.getCorners(FOCAL_PLANE):
+            xs.append(corner.x)
+            ys.append(corner.y)
+        xs.append(xs[0])
+        ys.append(ys[0])
+        xs = np.array(xs)
+        ys = np.array(ys)
+        rxs = nwRot[0, 0] * xs + nwRot[0, 1] * ys
+        rys = nwRot[1, 0] * xs + nwRot[1, 1] * ys
+        for ax in axes.ravel():
+            ax.plot(rxs, rys, c="k", lw=1, alpha=0.3)
+
+    fig.tight_layout()
+    if saveAs:
+        fig.savefig(saveAs)
+
+
+def makeAzElPlot(
+    fig: plt.Figure,
+    axes: np.ndarray[plt.Axes],
+    table: Table,
+    camera: Camera,
+    maxPoints: int = 1000,
+    saveAs: str = "",
+):
+    """Plot the PSFs on the focal plane, rotated to az/el coordinates.
+
+    Top left:
+        A scatter plot of the T values in square arcseconds.
+    Top right:
+        A quiver plot of e1 and e2
+    Bottom left:
+        A scatter plot of e1
+    Bottom right:
+        A scatter plot of e2
+
+    This function plots the data from the `table` on the provided `fig` and
```

Comment:
```suggestion
    This function plots the data from the ``table`` on the provided ``fig`` and
```
Variable and function-parameter names need double quotes, since they don't have a target to link to.  I'm sure I didn't catch all such cases of this.